### PR TITLE
feat: style formium components

### PR DESCRIPTION
**Pull request by csiyang**
# Description

### <samp>🤖 Generated by Copilot at c8babcf</samp>

This pull request adds and updates components for the `FormiumForm` component, which is used to render a form based on the Formium API. It also adds the code for the onboarding form page, which uses the `FormiumForm` component and handles authentication and app initialization. The changes improve the user interface, responsiveness, testability, and functionality of the form.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/34356604/todos/6561363409)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

### <samp>🤖 Generated by Copilot at c8babcf</samp>

*  Add the code for the onboarding form page that uses the FormiumForm component and the next-firebase-auth library ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-719483df49549a751a6b58ff274ddf6312157c1665900db7e79e213f88feb30eR1-R58))
*  Import and assign the icons for the buttons in the FormiumForm component based on the button text ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-f9823de2a4381d83835a9a449a757f9e843862fb7ad1b60fcd7220391bbb2d00R4-R7), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-f9823de2a4381d83835a9a449a757f9e843862fb7ad1b60fcd7220391bbb2d00R14))
*  Add the fullWidth, color, startIcon, and endIcon props to the Button component to style it according to the design ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-f9823de2a4381d83835a9a449a757f9e843862fb7ad1b60fcd7220391bbb2d00R21-R24))
*  Add the color prop to the MuiCheckbox component to style it according to the design ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-18979a638ae0c71c25aa8bdfccc021a69a6d0b874b6dcd8e8c85800f40c8ed52L9-R9))
*  Replace the Stack component with the Card and CardContent components to wrap the form elements in a card-like container ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-9f431ca7bb4865ab9f4434b43a6d0a119c4397a2f86f7dbaef7865948ff433ecL1-R2), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-9f431ca7bb4865ab9f4434b43a6d0a119c4397a2f86f7dbaef7865948ff433ecL14-R34))
*  Add the code for the FieldWrapper component that wraps each form field element with some padding ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-8b90137d569c4a0d17d55c9cd78bb1a0cb156bed05475e3237880925c3c874cdR1-R22), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-d9084c97812f70621601b8a4dc60d04fe3178517f50994aa818fc44e19f99a61R1), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-ce054a956c4bdcac15b7462e7088a49e4a0017757de209f0d9566727e38e5cdfR11), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-ce054a956c4bdcac15b7462e7088a49e4a0017757de209f0d9566727e38e5cdfL131-R143), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-e2e419b179c8247666bfff842d0c96b4e94d397225b6830fc57c2193c972112eR15), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-e2e419b179c8247666bfff842d0c96b4e94d397225b6830fc57c2193c972112eR36))
*  Add the data-testid attribute and the gap prop to the Box component in the FooterWrapper component for testing and spacing purposes ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-15d51310ed7bb4b87cdf85443645434e6ec1d51fca540b0ceaf746a1605a75abL12-R17))
*  Wrap the label and description elements in a Box component with some padding and a data-testid attribute in the FormControl component ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-9efbffe3bb0781a5b733a39e8e795735103422dcbd3b140dc71d408aac3ba18aR2), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-9efbffe3bb0781a5b733a39e8e795735103422dcbd3b140dc71d408aac3ba18aL20-R26))
*  Replace the Typography component with a Stack component that contains two Typography components to display the title and the subtitle of the header ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-1372929933c5118e1d5334f60b860d2494f1c41ca55204e9866074c8d1815c77L3-R7), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-1372929933c5118e1d5334f60b860d2494f1c41ca55204e9866074c8d1815c77L43-R54))
*  Replace the Paper component with the Container component to wrap the page content in a responsive container ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-f70c9b2f784e3e2dec58f2ed74aba4e6523a7b347df46d4d5bb8d36cb2a1e25bL1-R1), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-f70c9b2f784e3e2dec58f2ed74aba4e6523a7b347df46d4d5bb8d36cb2a1e25bL11-R15))
*  Remove the FormLabel component and the id prop from the RadioGroup component and add the color prop to the Radio component ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-4dc191f645382e6c01fbfc55ee016bdb296a41e1dfdb3e0e96b37c9b38da2b81L4), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-4dc191f645382e6c01fbfc55ee016bdb296a41e1dfdb3e0e96b37c9b38da2b81L11), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-4dc191f645382e6c01fbfc55ee016bdb296a41e1dfdb3e0e96b37c9b38da2b81L19-R23))
*  Modify the TextField component in the TextInput and Textarea components to create a filled input without a label and with a default width ([link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-b3c4ee51c91a9290b2e13e88c6a65b6d3840c50a18e8d3d892c77c0dd5ce6eedL11-R18), [link](https://github.com/JesusFilm/core/pull/1931/files?diff=unified&w=0#diff-ef67d1b80f1b21d9fc48be622afaab06d83014d3e0e438f5821b421c3fd4509dL7-R14))

### CODE REVIEW

File: apps/journeys-admin/pages/onboarding-form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { formiumClient } from '@core/shared/ui/formiumClient'
+import { FormiumForm } from '@core/shared/ui/FormiumForm'
+
+import { initAndAuthApp } from '../../src/libs/initAndAuthApp'
+
+interface OnboardingFormPageProps {
+  form: Form
+}
+
+function OnboardingFormPage({ form }: OnboardingFormPageProps): ReactElement {
```
Comment (csiyang) [inline apps/journeys-admin/pages/onboarding-form/index.tsx:19]: Will remove this page once style is approved

File: apps/journeys-admin/pages/onboarding-form/index.tsx
Comment (tataihono) [inline apps/journeys-admin/pages/onboarding-form/index.tsx:1]: refactor to use latest next-firebase-auth. please run npm i. AuthUser was refactored to just user in latest version.

File: libs/shared/ui/src/libs/FormiumProvider/FormiumProvider.tsx
Comment (tataihono) [inline libs/shared/ui/src/libs/FormiumProvider/FormiumProvider.tsx:1]: can we merge this with the formium form component itself? I don't imagine we'd ever want to consume the provider without the formium form itself.

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/pages/onboarding-form/index.tsx
Comment (csiyang) [reply apps/journeys-admin/pages/onboarding-form/index.tsx:1]: No need, this page is being removed

**Review by tataihono — APPROVED** [review]

**Pull request merged**